src/os/macos/etcpal/os_error.c

```diff
@@ -1,7 +1,5 @@
-/****
-    :  //
- github.com/ETCLabs/EtcPaletcpal_kEtcPalkEtcPalkEtcPalkEtcPalkEtcPalkEtcPalkEtcPalkEtcPalkEtcPalkEtcPalkEtcPalkEtcPalkEtcPalkEtcPalkEtcPalkEtcPalkEtcPalkEtcPalkEtcPalkEtcPalkEtcPalkEtcPalkEtcPalto_etcpal
```

Comment:
Any idea what this used to be for?  Was it some automation glitch?

Reply:
Oh it was almost certainly the VS Code Vim plugin, it has some issues...